strawberryfields/apps/subgraph.py

```diff
@@ -253,14 +286,19 @@ def resize(subgraph: list, graph: nx.Graph, min_size: int, max_size: int) -> dic
             grow_nodes = grow_subgraph.nodes()
             complement_nodes = nodes - grow_nodes
 
-            degrees = [
-                (c, graph.subgraph(list(grow_nodes) + [c]).degree()[c]) for c in complement_nodes
-            ]
-            np.random.shuffle(degrees)
+            degrees = np.array(
+                [(c, graph.subgraph(list(grow_nodes) + [c]).degree()[c]) for c in complement_nodes]
+            )
+            degrees_max = np.argwhere(degrees[:, 1] == degrees[:, 1].max()).flatten()
 
-            to_add = max(degrees, key=lambda x: x[1])
-            grow_subgraph.add_node(to_add[0])
+            if node_select == "uniform":
+                to_add_index = np.random.choice(degrees_max)
+            elif node_select == "weight":
```

Comment:
I'm confused. I thought node_select could be either `"uniform"` or a list/array. Where did we name the list/array option `"weight"`?